google/cloud/pubsub_v1/subscriber/_protocol/dispatcher.py

```diff
@@ -128,17 +131,46 @@ def dispatch_callback(self, items: Sequence[RequestItem]) -> None:
         nack_requests: List[requests.NackRequest] = []
         drop_requests: List[requests.DropRequest] = []
 
+        lease_ids = modack_ids = ack_ids = nack_ids = drop_ids = set()
```

Comment:
All these variables will refer to the same set object with this expression. See:

```
>>> x = y = set()
>>> x.add(1)
>>> x
{1}
>>> y
{1}
```

We prefer one line per variable definition in our style, too.